dmoj/settings.py

```diff
@@ -177,6 +179,7 @@
                         'judge.ProblemGroup',
                         'judge.ProblemType',
                         'judge.License',
+                        'judge.ProblemPointsVote',
```

Comment:
We don't expose comment votes in the sidebar, probably don't need to do so here.

Reply:
This was done to give access to `ProblemPointsVote.note`.

Reply:
Hmm, should we surface a link to a filtered admin view in the UI, or embed justifications in it? It seems like we can view problem vote distributions in the problem UI, but then an admin has to go to admin, `ProblemPointsVote` view, search by the same problem ID, and then read notes.

Reply:
Oh wait, this filtered link already exists...